doc/user/data/sql_funcs.yml

```diff
@@ -718,7 +718,11 @@
     url: /sql/types/jsonb#to_jsonb
 
 - type: Table
-  description: Table functions evaluate to a set of rows, rather than a single expression.
+  description: |
+    Table functions evaluate to a set of rows, rather than a single expression.
+    You can also add `WITH ORDINALITY` after the closing parenthesis of a table function call in a `FROM` clause.
+    This causes an `ordinality` column to appear, which is a 1-based numbering of the output rows of each call of
+    the table function. For example, `unnest(list) WITH ORDINALITY` numbers the unnested elements of each input list.
```

Comment:
Maybe?

```suggestion
    Table functions evaluate to a set of rows rather than a single expression.

    {{< annotation type="WITH ORDINALITY" >}}

    You can also add `WITH ORDINALITY` after the table function call in a `FROM`
    clause. This causes an `ordinality` column to appear, which is a 1-based
    numbering of the output rows of each call of the table function. 

    For example, `SELECT * FROM unnest(LIST[5,10]) WITH ORDINALITY AS
    list_item;` (or `SELECT * FROM unnest(LIST[5,10]) AS list_item WITH
    ORDINALITY;`) includes the `ordinality` column containing the 1-based
    numbering of the unnested elements.

    {{</ annotation >}}
```

Reply:
Thank you! I've made some further adjustments:
- "after the table function call" -> "after a table function call", because there can be multiple table function calls in a FROM clause.
- I've dropped the "(or `SELECT * FROM unnest(LIST[5,10]) AS list_item WITH
    ORDINALITY;`)", because this is not a syntax that I'd like people to use. I think no other system allows this. We used to allow only this syntax, but that was just a parser glitch.
- "of the unnested elements" -> "of the unnested items", to align the wording with the above `list_item` column name.

Reply:
Works for me!